src/kibana-cf_authentication/server/filterPath.test.js

```diff
@@ -0,0 +1,33 @@
+const each = require("jest-each").default
```

Comment:
We will need to install the `jest-each` module in the testing task in the `cg-deploy-logsearch` pipeline to pass to build the release.

Reply:
I was wondering about that. Do you know an easy way to do that?

Reply:
We can install in the task but will it add the dependencies to the input repository resource and pass them along to the build release step? Or does the build release get a fresh instances of the input repository resource?

Reply:
yeah, we need to move the test task to a new job so the dependencies will be cleared out before the build

Reply:
Job for installing and testing dev deps merged and pipeline updated https://github.com/cloud-gov/cg-deploy-logsearch/pull/252

Reply:
Thanks @apburnes !